dotCMS/src/main/webapp/html/js/dotcms/dojo/push/PushHandler.js

```diff
@@ -6,18 +6,32 @@ dojo.require("dotcms.dijit.RemotePublisherDialog");
 dojo.require("dotcms.dijit.AddToBundleDialog");
 dojo.require("dojox.data.QueryReadStore");
 dojo.require("dojox.data.JsonRestStore");
-
+dojo.require("dojo.NodeList-traverse");
 dojo.declare("dotcms.dojo.push.PushHandler", null, {
 
     assetIdentifier: "",
     dialog: null,
     title: "",
     bundleStore : null,
     environmentStore : null,
+    roleReadStore :null,
     user : null,
-    whereToSend : new Array(),
+    whereToSend : [],
     isBundle : false,
-    inialStateEnvs : new Array(),
+    inialStateEnvs : [],
+
+    //Optional params when a Workflow-action like behavior is desired.
+    workflow:null,
+    /*
```

Comment:
We can remove this if not used

Reply:
it is used. 
Take a look at : 
for example in the private function  _isWorkflowEnabled() also in function showWorkflowEnabledDialog
workflow is a property that is expected to be set externally. It holds WF the params.  
Also depending on its nullity the function remotePublish behaves as it was before or not.